client/app/vps/modal/shortcut/reverse-dns/vps-reverse-dns.controller.js

```diff
@@ -46,8 +46,10 @@ class VpsReverseDnsCtrl {
                 if (data && data.state) {
                     switch (data.state) {
                     case "ERROR" :
-                        this.CloudMessage.error(this.$translate.instant("vps_configuration_reversedns_fail"))
-                        break;
+                        this.CloudMessage.error(this.$translate.instant("vps_configuration_reversedns_fail"));
+                        const messages = !_.isEmpty(data.messages) ? data.messages : "";
```

Comment:
if a collection is empty, assign empty string on it?  Why not an empty array ([])?

Reply:
Thanks that's what I missing friday when I felt i need to leave ! :P

Reply:
done